src/fides/api/ops/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -76,7 +95,6 @@ def consent_request_verify_for_privacy_preferences(
         db.query(CurrentPrivacyPreference)
         .filter(CurrentPrivacyPreference.provided_identity_id == provided_identity.id)
         .order_by(CurrentPrivacyPreference.privacy_notice_id)
-        .all()
```

Comment:
I'm using the  `paginate` method from fastapi_pagination.ext.sqlalchemy instead of the one from `paginate` which is better for larger collections.  I don't need `all` here anymore